### PR TITLE
#159294934 Add project settings for multiple environments

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### What does this PR do?
Add Django settings to support multiple environments i.e. testing, development.
#### How should this be manually tested?
After cloning the project, the commands to run the project are python
* python manage.py runserver --settings=authors.settings.testing
* python manage.py runserver --settings=authors.settings.development

Using your terminal, add environment variables "DATABASE_URL" and "TEST_DATABASE_URL" and give them the necessary postgres urls.
#### What are the relevant pivotal tracker stories?
#159294934

### CODE REVIEW

File: authors/settings/development.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Database
+# https://docs.djangoproject.com/en/1.11/ref/settings/#databases
+
+from authors.settings.base import *
```
Comment (pluwum) [inline authors/settings/development.py:4]: What is this one importing exactly?

File: requirements.txt [hunk 2]
```diff
@@ -9,6 +9,7 @@ djangorestframework==3.8.2
 docopt==0.6.2
 idna==2.7
 psycopg2==2.7.5
+psycopg2-binary==2.7.5
```
Comment (pluwum) [inline requirements.txt:12]: Awesome!! thanks for updating this!!

File: authors/settings/development.py (hunk 1 above)
Comment (AGMETEOR) [reply authors/settings/development.py:4]: There's a base.py file in the settings folder that has a lot of the common stuff that's used by development.py and testing.py. So, I am importing all that e.g. INSTALLED APPS etc.

File: requirements.txt (hunk 2 above)
Comment (AGMETEOR) [reply requirements.txt:12]: Thank you.

File: authors/settings/development.py (hunk 1 above)
Comment (pluwum) [reply authors/settings/development.py:4]: @AGMETEOR I see one line in base.py and after the import, I don't see it used anywhere right now.

Comment (AGMETEOR) [reply authors/settings/development.py:4]: One line?

Comment (pluwum) [reply authors/settings/development.py:4]: @AGMETEOR , never mind. I have seen it!!

Comment (AGMETEOR) [reply authors/settings/development.py:4]: Alright.

**Pull request merged**